src/components/largePipelineWarning/largePipelineWarning.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../styles/variables';
+
+.warning {
+  position: absolute;
+  top: 30%;
+  left: 50%;
+  color: white;
+  font-family: 'Titillium Web', 'sans-serif';
+  text-align: center;
+  opacity: 0.85;
+
+  &__title {
+    padding: 16px 0px;
+    font-size: 38px;
+    line-height: 1.18;
+    letter-spacing: 0.27px;
+  }
+
+  &__subtitle {
+    width: 350px;
+    font-size: 1.48em;
+    line-height: 1.5;
+    opacity: 0.6;
+  }
+
+  &__btn {
```

Comment:
Might be best to be more specific with the name e.g. `render-button`

Reply:
yea I initially set this up under a similar naming as well, but in the end this still comes down to the class being a general button styling for the warning ( the class name is now `pipeline-warning__btn` with which it is descriptive enough to describe the button styling, hence I think it's still better to keep it as `__btn` .